CardsCashCasino/Data/PokerHand.cs

```diff
@@ -0,0 +1,25 @@
+﻿/*
+ *  Module Name: PokerHand.cs
+ *  Purpose: This module models the AI hands in the poker games.
+ *  Inputs: None
+ *  Outputs: None
+ *  Additional code sources: None
+ *  Developers: Mo Morgan
+ *  Date: 11/32024
+ *  Last Modified: 11/3/2024
+ *  Preconditions: None
+ *  Postconditions: None
+ *  Error/Exception conditions: None
+ *  Side effects: None
+ *  Invariants: None
+ *  Known Faults: None encountered
+ */
+
+namespace CardsCashCasino.Data;
+
+/// <summary>
+/// The <c>PokerHand</c> class represents the AI opponents' poker hands.
+/// </summary>
+public class PokerHand : CardHand
```

Comment:
I'd prefer if this name was a little more specific---instead of PokerHand, PokerAIHand or something that identifies it is just for the AI. We should be using "UserHand" for the user's hand in ALL games.